stonesoup/sensor/radar/tests/test_radar.py

```diff
@@ -1,14 +1,19 @@
 # -*- coding: utf-8 -*-
 import datetime
-
+from pytest import approx
 import numpy as np
 
-from ...functions import cart2pol
-from ...types.angle import Bearing
-from ...types.array import StateVector, CovarianceMatrix
-from ...types.state import State
 from ..radar import (
     RadarRangeBearing, RadarRotatingRangeBearing, RadarRasterScanRangeBearing)
+from ....functions import cart2pol
+from ....types.angle import Bearing
+from ....types.array import StateVector, CovarianceMatrix
+from ....types.state import State
+from ....types.groundtruth import GroundTruthState
+from ..radar import RadarRangeBearing, RadarRotatingRangeBearing, AESARadar
```

Comment:
This is causing a Flake8 error on testing, as some are also imported on lines 6-7 above. Should be squashed into single import line.

Reply:
Sorry I missed this. I have just pushed a corrected version.